src/App.tsx

```diff
@@ -17,18 +40,28 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <MemoTodoFilter
+                onSelect={query => setSelQuery(query)}
```

Comment:
```suggestion
                onSelect={setSelQuery}
```
No need to create additional function wrapper
Fix everywhere